requirements.txt

```diff
@@ -13,4 +11,4 @@ scipy==1.7.3
 # NUMBA
```

Comment:
I'm confused.  NUMBA is BSD-2, so shouldn't the comment be about the license to match everything else?

Also, why are we using cryptography here?

For plotting, I'd like to switch to plotnine, but that's going to involve bringing @gowerrobert around, so not yet.

Reply:
Those were added in 253329a32dcf279202e36f591676900ffdfbc2ed a long time ago. It looks like @gowerrobert may have just dumped his pip environment into the file. 

I didn't change anything besides delete fastaparser here, so I'm not too sure about the other contents